gui/src/components/History/HistoryTableRow.tsx

```diff
@@ -19,7 +25,7 @@ export function HistoryTableRow({
   sessionMetadata,
   index,
 }: {
-  sessionMetadata: SessionMetadata;
+  sessionMetadata: SessionMetadata | RemoteSessionMetadata;
```

Comment:
Perhaps worth renaming 

```suggestion
  sessionMetadata: LocalSessionMetadata | RemoteSessionMetadata;
```